packages/dappmanager/src/utils/timeout.ts

```diff
@@ -0,0 +1,31 @@
+// timestring does not have a @types package
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import timestring from "timestring";
+
+// Define build timeout (60 min)
+const defaultBuildTimeout = 60 * 60 * 1000;
```

Comment:
This default build timeout is for the SDK only. parseTimeout should return undefined in the default switch case